config/bidderinfo.go

```diff
@@ -246,6 +246,35 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 			infos[string(normalizedBidderName)] = info
 		}
 	}
+	return processAliasBidderInfo(infos)
+}
+
+func processAliasBidderInfo(infos BidderInfos) (BidderInfos, error) {
+	for bidderName, bidderInfo := range infos {
+		if len(infos[bidderName].AliasOf) > 0 {
```

Comment:
could directly use `bidderInfo` here

```
	for bidderName, bidderInfo := range infos {
           if bidderInfo.AliasOf == "" {
              continue
           }

           if err := validateAliases(bidderInfo, infos, bidderName); err != nil {
				return nil, err
	  }
          ....
```

Reply:
No specific reason to use `continue`. It was suggested to avoid nested `if` checks. 
Changes from https://github.com/prebid/prebid-server/commit/d54235565e2b29bff0f85cfa87726c031a409cef seems fine